components/atoms/Accordion/Accordion.js

```diff
@@ -0,0 +1,55 @@
+import propTypes from "prop-types";
```

Comment:
Also, one last comment is that none of the other components have an extra directory here (`/atoms/Accordion`), so we should probably move it into the base `atoms` directory unless we had a reason to keep it separate.

Reply:
no, just cleaner in my opinion